windows.js

```diff
@@ -8,6 +8,12 @@ const exec = require('@actions/exec')
 const tc = require('@actions/tool-cache')
 const rubyInstallerVersions = require('./windows-versions').versions
 
+// needed for 2.2, 2.3, and mswin, cert file used by Git for Windows
+const certFile = 'C:\\Program Files\\Git\\mingw64\\ssl\\cert.pem'
```

Comment:
I hope this location will remain available, it sounds likely enough for now.

Reply:
Git for Windows has always been there.  It's the git that's used by all the actions and any Windows devs using git.  There are also GitHub/MSFT staff involved with it.  IOW, its location is probably more locked down than the locations of the toolcache Rubies...